modules/server-endpoints/language-server/src/main/java/org/wso2/carbon/identity/developer/lsp/completion/FunctionLibraryManager.java

```diff
@@ -0,0 +1,83 @@
+package org.wso2.carbon.identity.developer.lsp.completion;
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import org.wso2.carbon.identity.functions.library.mgt.FunctionLibraryManagementService;
+import org.wso2.carbon.identity.functions.library.mgt.FunctionLibraryManagementServiceImpl;
+import org.wso2.carbon.identity.functions.library.mgt.exception.FunctionLibraryManagementException;
+import org.wso2.carbon.identity.functions.library.mgt.model.FunctionLibrary;
+import java.util.HashMap;
+import java.util.List;
+
+public class FunctionLibraryManager {
```

Comment:
Need class comment, explaining what this supposed to do, how it should be used, and when, etc.